src/plugins/postgres_table_copy_operator.py

```diff
@@ -112,6 +116,50 @@ def __init__(
                 "even though source table will be dropped."
             )
 
+    def _drop_tables_if_unequal(self, cursor: Any, all_table_copies: List[TableMapping]) -> None:
+        """Drop the target tables that are unequal to their source table."""
+        for table_mapping in all_table_copies:
+            cursor.execute(
+                sql.SQL(
+                    """
+              (SELECT data_type,
+                      ordinal_position,
+                      column_name
+               FROM information_schema.columns
+               WHERE table_name = %(source_table_name)s
+               EXCEPT SELECT data_type,
+                             ordinal_position,
+                             column_name
+               FROM information_schema.columns
+               WHERE table_name = %(target_table_name)s)
+            UNION ALL
+              (SELECT data_type,
+                      ordinal_position,
+                      column_name
+               FROM information_schema.columns
+               WHERE table_name = %(target_table_name)s
+               EXCEPT SELECT data_type,
+                             ordinal_position,
+                             column_name
+               FROM information_schema.columns
+               WHERE table_name = %(source_table_name)s)
+            """
+                ),
+                (
+                    {
+                        "source_table_name": table_mapping.source,
+                        "target_table_name": table_mapping.target,
+                    }
+                ),
+            )
```

Comment:
Why the extra parentheses?

Reply:
Ah, that was a left-over from editing stuff, removed! Tx.